oktapam/resource_ad_task_settings.go

```diff
@@ -194,16 +196,19 @@ func resourceADTaskSettingsRead(ctx context.Context, d *schema.ResourceData, m i
 			_ = d.Set(key, value)
 		}
 	} else {
-		logging.Infof("ADTaskSettings %s does not exist", id)
+		logging.Infof("ADTaskSettings %s does not exist", adTaskSettingsId)
 	}
 
 	return diags
 }
 
 func resourceADTaskSettingsUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	c := m.(client.OktaPAMClient)
-	taskSettingsId := d.Id()
-	adConnId := d.Get(attributes.ADConnectionID).(string)
+
+	adConnId, adTaskSettingsId, err := parseADTaskSettingsResourceID(d.Id())
```

Comment:
Same here (and s/Id/ID/)

Reply:
Fixed in [6c0355a](https://github.com/okta/terraform-provider-oktapam/pull/65/commits/6c0355a9175e980f2658c6d23769aa347eac8d8f)